cloud/aws/ecs.md

```diff
@@ -3,7 +3,9 @@ layout: page
 title: "Amazon ECS with Portworx"
 keywords: portworx, amazon, docker, aws, ecs, cloud
 sidebar: home_sidebar
-redirect_from: "/portworx-on-ecs2.html"
+redirect_from:
+  - /portworx-on-ecs2.html
+  - /cloud/aws-ecs.html

```

Comment:
There's also a `portworx-on-ecs.md` file at the root of this repo, should this also get directed here?

Reply:
I think the file should be removed if it's unreferenced for sure (and new redirect)